### PR TITLE
[Core][Hotfix] Missing abs did quadratic extrapolation not work properly

### PR DESCRIPTION
**Pull request by loumalouomega**
Fix problem found in #4710. There was a  missing abs, the average area may be negative for quadratic formulations. Adding test

### CODE REVIEW

**Comment by philbucher** [comment]
thanks!

@ZhimingGuo can you test this please?

**Comment by loumalouomega** [comment]
@philbucher you forgot to approve

**Comment by philbucher** [comment]
I would first like to see if it works now for @ZhimingGuo

**Comment by loumalouomega** [comment]
@ZhimingGuo can you confirm that now works better?

**Comment by ZhimingGuo** [comment]
not really, it is still not so smoothly.
vonmises

**Comment by loumalouomega** [comment]
> not really, it is still not so smoothly.
> vonmises

I think it is related with the average value (NODAL_AREA), certantly I don't know how to improve it. And after looking to the books of Felippa I don't have any idea what is exactly wrong apart from this.

**Comment by loumalouomega** [comment]
In any case current PR improves current behaviour, we can try to find a finer solution later...

File: kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp [hunk 1]
```diff
@@ -205,5 +267,49 @@ namespace Kratos
...
+            Model this_model;
+            ModelPart& r_model_part = this_model.CreateModelPart("Main", 2);
+            ProcessInfo& r_current_process_info = r_model_part.GetProcessInfo();
+            r_current_process_info[DOMAIN_SIZE] = 3;
+
+            r_model_part.AddNodalSolutionStepVariable(DISPLACEMENT);
+
+            // In case the StructuralMechanicsApplciation is not compiled we skip the test
+            if (!KratosComponents<ConstitutiveLaw>::Has("LinearElastic3DLaw"))
+                return void();
```
Comment (philbucher) [inline kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:286]: why not just `return`?

**Review by philbucher — APPROVED** [review]
approving as intermediate solution, thx

File: kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp (hunk 1 above)
Comment (loumalouomega) [reply kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:286]: I think I had a compilation problem

**Pull request merged**